scripts/build.sh

```diff
@@ -18,10 +18,10 @@ cd "$DIR"
 BUILD_TAGS="${BUILD_TAGS:-"vault"}"
 
 # Get the git commit
-GIT_COMMIT="$(git rev-parse HEAD)"
+GIT_COMMIT="$("$SOURCE_DIR"/crt-builder.sh revision)"
```

Comment:
Love this 😍 !  It allows us to remove the complex logic to determine where this is being called from 👍